### PR TITLE
Code Review Request 2 For Ruby Project 3 Tic Tac Toe Game

### PR DESCRIPTION
**Pull request by amiraabouhadid**
We were asked to create the user interface by editing only the main  file such that players are welcomed then asked for their names and their respective tokens.  Each player is then asked to enter a digit from 1-9 to indicate the unfilled position at which he/she wish to place their token during their respective turns. An invalid move is flagged and indicated to the players and the players are asked to enter another digit. When either player wins or they tie, a GAME OVER is displayed with the game's results and the game is reset.

### CODE REVIEW

**Review by DeliceLydia — COMMENTED** [review]
# Status: Invalid image

Hello Team Good job so far but I have to make this PR invalid because
the app is not running when I run ruby bin/main.rb please make sure that you can play the
game before submitting

Best,

Lydia Ingabire

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    @board.show
+    puts 'Welcome to Ruby\'s Tic Tac Toe!'
+    player1 = Player.new(name('X'))
+    player2 = Player.new(name('O'))
+    @game = Game.new(player1, player2)
+    game_round
+  end
+
+  def name(token)
+    puts 'Please enter player name:'
```
Comment (pacyL2K19) [inline bin/main.rb]: Nice to create a method handling this action, a small improvement on can be to not only ask them with the same question but to put a meaningful message like `First player, please enter your name and then the second player  ....`

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    name = gets.chomp
+    name(token) if name.empty?
+    name
+  end
+
+  def game_round
+    loop do
+      @board = Board.new
+      @board.show
```
Comment (pacyL2K19) [inline bin/main.rb:130]: Like this line :smiley:

**Review by pacyL2K19 — CHANGES_REQUESTED** [review]
# Status: Changes Required :stop_sign: 
Hello @amiraabouhadid  you made a great work so far, your code looks great and I enjoyed reading it :smiley: 
> 
However, there are some updates you need to make before the milestone get approved
- [x] The game is supposed to run 9 times or more before ending with a simple message like `It's a drawing move` from now, you don't have to compete with the game logic, your project stops after few moves and assigns the winner badge to a wrong one see screenshot below 
wrong1
> 
KIndly work on these changes before submitting them again 
Feel free to reach out to me on slack if any question
Happy coding :desktop_computer:

**Review by sinansevgi — APPROVED** [review]
# Status: Approved! ✅ 
----
Congratulations! 
----
Good job implementing the change as requested by the previous TSE.
 👏I can see you've implemented the README file on this milestone. Congrats. 
You can now merge and continue with the next milestone.
----
### Sinan Sevgi

**Pull request merged**